js/ui/camera.js

```diff
@@ -556,7 +556,7 @@ class Camera extends Evented {
      *     It does not correspond to a fixed physical distance, but varies by zoom level.
      * @param {number} [options.screenSpeed] The average speed of the animation measured in screenfuls
      *     per second, assuming a linear timing curve. If `options.speed` is specified, this option is ignored.
-     * @param {Object} [eventData] Data to propagate to any event listeners.
+     * @param {Object} [eventData] Data to propagate to any event listeners. Properties on the object will be copied onto the event argument.
```

Comment:
These two sentences are each trying to convey the same bit of information. Do you think we could rewrite to convey it clearly and succinctly in one?

`Additional [event data]{@link Evented#fire} to be included with all events triggered by this method`